mep/people/templates/people/member_cardlist.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+{% load render_bundle from webpack_loader %}
+{% load static mep_tags %}
+
+{% block page-subtitle %}{{ member.firstname_last }} Cards · Library Members · {% endblock %}
+
+{% block js %}
+{% render_bundle 'activities' 'js' %}
+{% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.firstname_last style="detail" %}
+{% endblock %}
+
+{% block main-style %}tabbed white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+<nav class="tabs" aria-label="sections">
+    <li class="tab">
+        <a href="{% url 'people:member-detail' slug=member.slug %}">biography</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">activities</a>
+    </li>
+    <li class="tab" aria-selected="true">
+        <span>cards</span>
+    </li>
+</nav>
+
+<section class="member-cards">
+    <ol class="card-results">
+    {% for card in cards %}
+    <li class="card result">
+        {# <a href="#" aria-label=""> #}
+                {# card thumbnails and year if known #}
+                <picture>
+                  {% with card.image|iiif_image:"size:width=225" as 1xthumbnail %}
+                    <source srcset="{{ 1xthumbnail }}, {{ card.image|iiif_image:"size:width=450" }} 2x">
+                    <img src="{{ 1xthumbnail }}" alt="">
+                  {% endwith %}
+                </picture>
+                {% with card.footnote_set.event_date_range as dates %}
+                <div class="card-dates {% if not dates %}unknown{% endif %}">
+                    <dt>Card Years</dt>
```

Comment:
the only allowed parents for `<dt>` and `<dd>` are `<dl>` (or `<div>`s inside `<dl>`s). since this also isn't a description _list_ (there's only one pair of key/value) I think it's fine to just render them as spans.

Reply:
alternatively you can use the `<time>` element, like in the content page attribution snippets

Reply:
SRs _should_ read something like "link: card years, 1923" when the link is focused, because the alt is blank so the image will be skipped. i think this is what we want. would test it right now but my voiceover is being belligerent and not starting >:(

Reply:
Thanks, this is useful. I don't think `<time>` works because in some cases we have a range.